adoc/admin-stratos-web-console.adoc

```diff
@@ -0,0 +1,233 @@
+= Stratos Web Console
+
+The Stratos user interface (UI) is a modern web-based management application for
+{kube} and for Cloud Foundry distributions based on {kube} like SUSE
+Cloud Application Platform.
+
+Stratos provides a graphical management console for both developers and system
+administrators.
+
+A single Stratos instance can be used to monitor multiple {kube} clusters
+as long as it is granted access to their {kube} API endpoint.
+
+If you plan to deploy SUSE Cloud Application Platform on your {productname}
+cluster please skip this section of the documentation and refer
+to the official SUSE Cloud Application Platform instructions.
+
+== Introduction
+
+This document aims to describe how to install Stratos in a {productname} cluster
+that doesn't plan to run any SUSE Cloud Application Platform components.
+
+The Stratos stack is deployed using a helm charts and consists of its web
+UI POD and a MariaDB one that is used to store configuration values.
+
+== Prerequisites
+
+=== Persistent Storage
+
+The MariaDB instance used by Stratos requires a persistent storage to store
+its data.
+
+The cluster must have a {kube} Storage Class defined.
+
+=== Exposing the Web UI
+
+The web interface of Stratos can be exposed either via a Ingress resource or
+by using a Service of type LoadBalancer or even both at the same time.
```

Comment:
Should `NodePort` be another option?

Reply:
Yes it is, but it's not really flexible... Everybody usually goes with either Ingress or LoadBalancer for common workload deployments